lib/common/template.js

```diff
@@ -108,5 +108,18 @@ function templateServiceFactory(FileLoader, Constants, Promise, _, waterline, Lo
         });
     };
 
+    TemplateService.prototype.unlink = function (name, scope) {
+        scope = scope || 'global';
+        return new Promise(function(resolve, reject) {
```

Comment:
Interesting, could be a bug or some weird behavior with waterline, let me try to repro.